### ISSUE TITLE
Issue with k8s.io/docs/concepts/cluster-administration/cloud-providers/#cloudstack

### ISSUE THREAD

**Issue by onitake** [kubernetes/website#14817]
**This is a Bug Report**

**Problem:**
The section that documents integration with CloudStack environments is incorrect.
It links to the OpenStack cloud provider, which has nothing to do with CloudStack.

**Proposed Solution:**
There is no _official_ replacement for the old CloudStack cloud provider in Kubernetes, but a number of unofficial providers exist.

I would suggest linking to the CCM that was built by my colleagues and myself. It is based on the original cloud provider code in Kubernetes. We also added a few improvements that were not accepted into k8s due to the deprecation.

The source code is here: https://github.com/swisstxt/cloudstack-cloud-controller-manager

**Page to Update:**
https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/

Kubernetes Version: 1.13.3

Additional Information:

Proposed replacement text:
```
## CloudStack

An external cloud controller manager that should work with most CloudStack environments is available here: [swisstxt/cloudstack-cloud-controller-manager](https://github.com/swisstxt/cloudstack-cloud-controller-manager)

### Node Name

The CloudStack cloud controller manager uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object. To make this work, the pod running cloudstack-cloud-controller-manager needs to have a pod security policy that allows fetching the node name.

Note that the Kubernetes Node name must match the CloudStack VM name exactly.
```

**Comment by sftim** [comment]
Thanks for the feedback
/language en

**Comment by onitake** [comment]
There is now an official Apache project for a CloudStack k8s provider: https://github.com/apache/cloudstack-kubernetes-provider

I will prepare a PR shortly that changes the documentation to point to that.

**Comment by onitake** [comment]
Already fixed by #16129 - closing.
